src/containers/edit-profile/professional-info-tab/add-professional-category-modal/AddProfessionalCategoryModal.tsx

```diff
@@ -115,12 +150,22 @@ const AddProfessionalCategoryModal: FC<AddProfessionalCategoryModalProps> = ({
     })
   }
 
+  const handleSubjectDelete = (id: string) => {
+    handleDataChange({
+      subjects: data.subjects.filter((el) => el._id !== id)
+    })
+  }
+
+  const handleBlockOption = (option: CategoryNameInterface) => {
+    return (
+      blockedCategoriesOptions.some(
+        (mainSubject) => mainSubject.category._id === option._id
+      ) && option._id !== data.category._id
```

Comment:
Split condition with `some` and other condition into separate variables for better readability

Reply:
Thank you fixed!